.github/workflows/release-v2.yml

```diff
@@ -59,7 +59,7 @@ jobs:
         id: create_release
         uses: softprops/action-gh-release@v1
         with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Lovely. So there's very little standardization/common practices in naming of configuration properties across actions :-(

Reply:
I tried to test using that `act` tool using the dryrun mode but it didn't pick up on the error.